internal/serverinstall/k8s.go

```diff
@@ -1,17 +1,15 @@
 package serverinstall
 
 import (
-	"bytes"
 	"context"
 	json "encoding/json"
 	"fmt"
 	"io/ioutil"
 	"net"
-	"os/exec"
 	"strings"
 	"time"
 
-	"gopkg.in/yaml.v2"
+	"github.com/ghodss/yaml"
```

Comment:
Is there a reason we changed yaml packages here?

Reply:
Yeah, I resolved the comment for how I got here on this PR, but basically the client-go package for k8s needs its own special way to unmarshall YAML. This yaml package actually seems to do a decent job for what we want without any other major code changes from what it is now, which is just calling unmarshall on the byte array into a Secrets struct. Otherwise if that isn't agreeable, there's a more complicated way to do this with the k8s api package.